src/rct/manifest_commands.py

```diff
@@ -270,6 +270,11 @@ def _print_products(self, zip_archive):
 
             self._print_section(_("Provided Products:"), sorted(to_print), 2, False)
 
+            # Get the derived provided Products (if available)
+            if data["pool"]["derivedProvidedProducts"]:
```

Comment:
Caused a failing test: `KeyError: 'derivedProvidedProducts'`; I think

    if "derivedProvidedProducts" in data["pool"]:

will do it.